astro/src/content/docs/get-started/start-here/index.mdx

```diff
@@ -0,0 +1,196 @@
+import StartDiagram from 'src/diagrams/docs/get-started/start-here/launch-delegate-add.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import Aside from 'src/components/Aside.astro';
+import InlineField from 'src/components/InlineField.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+import DockerComposeFiles from 'src/content/docs/get-started/download-and-install/_docker-compose-files.mdx';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+import DockerSpinup from 'src/components/quickstarts/DockerSpinup.astro';
+import DockerRequirements from 'src/components/quickstarts/DockerRequirements.astro'
+import DockerInstallAndStart from 'src/components/quickstarts/DockerInstallAndStart.astro'
+import GitPodStart from 'src/components/quickstarts/GitPodStart.astro'
+
+To get started with FusionAuth, you need to do three things:
+
+* Launch FusionAuth
+* Delegate authentication
+* Add functionality to your application
+
+<StartDiagram alt="Launch, then delegate, then add functionality." />
+
+Let's walk through an example where you'll add a Login With Google button to an application with no code changes required.
+
+## Prerequisites
+
+* git
+* Docker (if you want to follow the Docker instructions)
+* Github account (if you choose to use GitPod instead)
+
+## Launch FusionAuth
+
+FusionAuth is downloadable software, so you need to run it somewhere. For this guide, you can either
+
+* Use Docker to install and run FusionAuth and the application
+* Use GitPod to run FusionAuth in the cloud in your GitPod account, with no software install needed.
+
+Expand the section you want to follow and get started.
+
+<details>
+<summary>Docker</summary>
+
+<DockerRequirements />
+<DockerInstallAndStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+</details>
+
+
+<details>
+<summary>GitPod</summary>
+
+<GitPodStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+
+</details>
+
+### Delegate Authentication
+
+The next step is to delegate authentication for your application, and verify that the change is applied.  For this you will:
+* Add the admin@example.com user to the application
+* Logout from the administrative user
+* Go to the test application
+* Login as the administrative user
+
+### Step 1. Bring up the administrative interface
+
+In the previous step, you logged in to the system as an admin user.  Go back to that browser screen, which should be showing the administrative interface for FusionAuth.
+
+### Step 2. Choose the correct user
+
+* Navigate to "Users" in the left-hand sidebar (you may need to use the hamburger icon at the top left of the screen to see all options)
+* Choose the "admin@example.com" user (Dinesh Chugtai)
+* Choose "Manage" from the list of actions
+
+### Step 3. Add registration to the application
```

Comment:
Is it really important to number the steps like this? I don't think we do that anywhere else.